lib/components/InputGroup.tsx

```diff
@@ -32,15 +32,15 @@ export const InputGroup = ({
   }, [expanded])
 
   return (
-    <div className={tx('p-4 flex flex-col gap-y-4 bg-white rounded-xl', className)}>
+    <div className={clsx('col gap-y-4 p-4 bg-white rounded-xl', className)}>
       <div
-        className={tx('flex flex-row justify-between items-center', {
+        className={clsx('row justify-between items-center', {
           'cursor-pointer': isExpandable && !disabled,
           'cursor-not-allowed': disabled,
         },
         {
-          'text-hw-primary-400': !disabled,
-          'text-hw-primary-200': disabled
+          'text-primary': !disabled,
+          'text-primary/40': disabled
```

Comment:
Is this valid?

Reply:
Yes, its the [tailwind opacity operator](https://tailwindcss.com/docs/colors#adjusting-opacity)